test/libsolidity/ASTJSON/assembly/call.json

```diff
@@ -129,7 +129,8 @@
                 "nodeType": "InlineAssembly",
                 "src": "47:43:1"
               }
-            ]
+            ],
+            "unchecked": false
```

Comment:
Should we just skip it if it is false?

Reply:
I wonder whether - despite that that would diverge from the C++ AST - we should instead introduce a node type ``Unchecked`` or ``UncheckedBlock`` instead...
I actually haven't read the PR yet: would this be ``false`` in nested (regular) blocks inside an unchecked block?

Reply:
Yes, it would be false in nested regular blocks. Maybe you are right, I'll change the node type instead.